cuML/src/solver/solver_c.h

```diff
@@ -70,5 +70,42 @@ void sgdPredictBinaryClass(const float *input, int n_rows, int n_cols, const flo
 void sgdPredictBinaryClass(const double *input, int n_rows, int n_cols,
 		const double *coef, double intercept, double *preds, int loss);
 
+
+void cdFit(float *input,
+		   int n_rows,
+		   int n_cols,
+		   float *labels,
+		   float *coef,
+		   float *intercept,
+		   bool fit_intercept,
+		   bool normalize,
+		   int epochs,
+		   int loss,
```

Comment:
I agree with the above, a lambda or class, or even an enum could be used here to make this easier to use (and possibly extend).

Reply:
Need help on the cython side for enum. Lamba input would be nice but needs more work. Maybe, we can add lambda to other algorithms as well in the future releases.